index.html

```diff
@@ -88,5 +88,301 @@ <h3>LET’S CONNECT</h3>
         </div>
       </div>
     </section>
+    <section class="works">
+      <div class="grid works-grid-container wrap padding-top-0">
+        <div class="flex works-grid-item">
+          <div class="work-image">
+          <img
+            class="width-100"
+            src="Image/Tonic.png"
+            alt="Calender"
+          />
+          </div>
+          <div class="flex section-title-wrapper">
+          <h1 class="section-title margin-top-12">Tonic</h1>
+          <div class="work-period">
+            <ul>
+              <li class="text-color-primary">Canopy</li>
+              <li>Back End Dev</li>
+              <li>2015</li>
+            </ul>
+          </div>
+          <p class="work-description text-color-primary margin-top-20">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="work-categories margin-top-12 padding-0">
+            <li><span>html</span></li>
+            <li><span>css</span></li>
+            <li><span>javascript</span></li>
+          </ul>
+          <div>
+            <button type="button" class="button button-enabled">
+            See Project
+          </button>
+          </div>
+        </div>
+      </div>
+      <div class="flex works-grid-item">
+        <img
+          class="width-100"
+          src="Image/Calender.png"
+          alt="Fitness screenshot"
+        />
+        <div class="flex work-details-wrapper">
+        <h1 class="section-title margin-top-12">Multi-Post
+          <br> Stories
+        </h1>
+        <div class="work-period">
+          <ul>
+            <li class="text-color-primary">Canopy</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+        <p class="work-description text-color-primary margin-top-20">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="work-categories margin-top-12 padding-0">
+          <li><span>html</span></li>
+          <li><span>css</span></li>
+          <li><span>javascript</span></li>
+        </ul>
+        <div>
+          <button type="button" class="button button-enabled">
+          See Project
+        </button>
+        </div>
+      </div>
+      </div>
+    </div>
+  </section>
+  <section class="works">
+    <div class="grid works-grid-container wrap padding-top-0">
+      <div class="flex works-grid-item">
+        <img
+          class="width-100"
+          src="Image/Fitness.png"
+          alt="portfolio screenshot"
+        />
+        <div class="flex work-description-wrapper">
+        <h1 class="section-title margin-top-12">Tonic</h1>
+        <div class="work-period">
+          <ul>
```

Comment:
- [ ] Your individual project cards are not centered as shown in the Figma design. Kindly center the project cards to achieve a more consistent design 👍 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/185635985-ef0fba66-6636-4716-be19-1fc086e178b3.png)

### Figma Design
![image](https://user-images.githubusercontent.com/46839548/185635629-60ba1bb0-101b-4dc5-8b43-4d1507453497.png)